### PR TITLE
Clarify Resource Authorization is a refinement of WAC/Solid

### LINKED ISSUE #101: Clarify WebAC section
The WebAC section includes this statement

> Implementations must support extended control relationships in which the access control resource for some resource has its own access control resource and so on.

What does that mean? The phrase "extended control" does not appear in the Solid WebAC specification nor in the W3C wiki page for WebAC. And how does that differ from what is already defined as the `acl:Control` mode that is [already clearly defined](https://github.com/solid/web-access-control-spec#referring-to-the-acl-resource-itself) in the Solid WebAC specification?

The WebAC section also has this statement:

> Implementations must make access control resources discoverable via the Link: rel="acl" HTTP header, as discussed [here](https://github.com/solid/web-access-control-spec#acl-resource-location-discovery).

Referring to remote resources using a "here link" is generally bad practice. And if the Fedora specification is simply recapitulating what the Solid WebAC specification already states, why is this statement in the Fedora specification?

This also applies to this statement:

> 6.1 Inheritance §
> An implementation must respect [LDP]-containment for access control, as described [here](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm).

Why is the Fedora specification recapitulating statements from other specifications that it already requires?

### PR DESCRIPTION
**Pull request by barmintor**
fixes #101

### CODE REVIEW

**Review by ajs6f — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -584,23 +584,18 @@ <h2 class='informative'>Implementation Patterns</h2>
...
-        MUST support extended control relationships in which the access control resource for some
-        resource has its own access control resource and so on. Implementations MUST make access
-        control resources discoverable via the <code>Link@rel="acl"</code> HTTP header, as discussed
-        <a href="https://github.com/solid/web-access-control-spec#acl-resource-location-discovery">here</a>.
-      </p>
-
+      <section id="solid-ldp-acls">
+        <h2>ACLs are LDP RDF Sources</h2>
+        <p>
+          The linked Access Control List Resource (ACL) for a controlled resource by a conforming server MUST itself be a LDPRS. This ACL resource SHOULD be located in the same server as the controlled resource.
```
Comment (acoburn) [inline index.html]: nit picking, but it's nice to wrap the lines at some sensible length to make reading them easier. (There are other advantages, too!)

Comment (barmintor) [inline index.html]: Needed to rebase, but corrected now. Thanks!

**Review by acoburn — APPROVED** [review]

**Comment by escowles** [comment]
👍

**Comment by zimeon** [comment]
:+1:

File: index.html [hunk 2]
```diff
@@ -712,23 +712,23 @@ <h2 class='informative'>Vary</h2>
...
         To configure access control restrictions, implementations MUST follow the recommendations
-        of [[!SOLIDWEBAC]].
+        of [[!SOLIDWEBAC]] with the following additional requirements:
       </p>
-      <p>
-        Implementations supporting access control MUST use the [[!WEBAC]] model to
-        configure that functionality. Any access control resource in use MUST be a <a>LDP-RS</a>,
-        which SHOULD be located in the same server as the controlled resource. Implementations
-        MUST support extended control relationships in which the access control resource for some
-        resource has its own access control resource and so on. Implementations MUST make access
```
Comment (awoods) [inline index.html:722]: In order to ensure the use of `Link: rel="acl"`, I would prefer to keep the follow language in the spec:
> Implementations MUST make access control resources discoverable via the <code>Link: rel="acl"</code> HTTP header, as discussed <a href="https://github.com/solid/web-access-control-spec#acl-resource-location-discovery">here</a>

**Review by awoods — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (acoburn) [inline index.html]: Yes, but please change the formatting. A "click here" link is really not best practice.

Comment (barmintor) [reply index.html:722]: I added a new section to (re)capture this requirement.

**Review by awoods — APPROVED** [review]

**Comment by awoods** [comment]
@escowles / @zimeon : If you are ok with the recent updates, I suggest we move this PR forward.

**Comment by escowles** [comment]
@awoods Yes, I am still 👍  on this PR.

**Comment by barmintor** [comment]
@dannylamb ?

**Review by dannylamb — APPROVED** [review]

**Comment by dannylamb** [comment]
Im good with this :+1:

**Review by zimeon — APPROVED** [review]

**Pull request merged**